src/test/java/org/zalando/nakadi/controller/SchemaControllerTest.java

```diff
@@ -7,28 +7,35 @@
 import org.mockito.Mockito;
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.context.request.NativeWebRequest;
+import org.zalando.nakadi.domain.EventType;
+import org.zalando.nakadi.service.EventTypeService;
 import org.zalando.nakadi.service.Result;
 import org.zalando.nakadi.service.SchemaService;
 import org.zalando.problem.Problem;
 
 import javax.ws.rs.core.Response;
 
+import static org.zalando.nakadi.utils.TestUtils.buildDefaultEventType;
+
 public class SchemaControllerTest {
 
     private SchemaService schemaService;
     private NativeWebRequest nativeWebRequest;
+    private EventTypeService eventTypeService;
 
     @Before
     public void setUp() {
         schemaService = Mockito.mock(SchemaService.class);
         nativeWebRequest = Mockito.mock(NativeWebRequest.class);
+        eventTypeService = Mockito.mock(EventTypeService.class);
     }
 
     @Test
     public void testSuccess() {
         Mockito.when(schemaService.getSchemas("et_test_name", 0, 1)).thenReturn(Result.ok(null));
         final ResponseEntity<?> result =
-                new SchemaController(schemaService).getSchemas("et_test_name", 0, 1, nativeWebRequest);
+                new SchemaController(schemaService, eventTypeService)
```

Comment:
@lmontrieux Now I realised something. We use MockMVC for testing controllers. We do that because it tests some other functionality that is left behind when interacting with controller directly. Please take a look at some other controller class for examples on how to use it.